spec/unit/azurerm_base_spec.rb

```diff
@@ -98,144 +97,50 @@ def validate_cert()
 		end
 	end
 
-	describe "azure base tests - for azure profile" do
```

Comment:
@adamedx,
the azurerm_base_spec file was created during current PR. the test cases which i removed were incorrect, they were related to ASM. i had copied basic sanity checks from asm_base file, however the checks which i removed where not needed for arm and the methods they were trying to mock doesn't exist in azurerm_base file